src/v/datalake/translation/translation_probe.cc

```diff
@@ -97,6 +97,77 @@ void translation_probe::register_created_files_metrics() {
       });
 }
 
+void translation_probe::register_throughput_metrics() {
+    namespace sm = ss::metrics;
+    std::vector<sm::label_instance> labels{
+      namespace_label(_ntp.ns()),
+      topic_label(_ntp.tp.topic()),
+      partition_label(_ntp.tp.partition()),
+    };
+    _public_metrics->add_group(
+      group_name,
+      {{
+         sm::make_counter(
+           "raw_bytes_processed",
```

Comment:
I am not sure we need the processed vs success distinction given the cost of metrics, but I guess it's fine 🤷

Reply:
Yea that's fair. Removed the success case since i think processing is more useful to know about for debugging